<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAccount</class>
 <widget class="QDialog" name="CreateAccount">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Desk Account</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color:  #C2272D;
}

QDialog{
	background-color:  #C2272D;
}

QLineEdit{
	background-color: #fff;
	color: #000000;
	border-radius:  15px;
	padding-left : 11px;
}

QPushButton{
background-color: #fff;
text-transform: uppercase;
border-radius:  15px;
color: #000000;

}

QToolButton
{
    border-radius: 30px;
}

QMenu{
background-color:  gray;
color: #fff;
}

QMenuBar{
background-color:  #434343;
color: #fff;
}

QAction{
background-color: gray;
color: #fff;

}

QMenuBar::item:selected{
background-color: gray;
color: #fff;
}

QMenu::item:selected{
background-color:  #404040;
}

QMessageBox{
background-color:  #fff;
}
</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>571</width>
     <height>531</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>571</width>
     <height>531</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>571</width>
     <height>531</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="keyinputLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: #fff;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;9999 used (ignore; used for testing)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstnameInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Firstname</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ageInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox{
background-color: #fff;
color:  #000000;

}</string>
    </property>
    <property name="maximum">
     <number>130</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordInput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyInput">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>key - 9999</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="repeatpasswordInput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Repeat Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="windownameLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
text-transform: uppercase;
color: #fff;
line-height: 5px;
color:  #fff;

}</string>
    </property>
    <property name="text">
     <string>Create an Account</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastnameInput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Lastname</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
